### PR TITLE
docs: add note about renames of Room and Furniture

### PR DESCRIPTION
**Pull request by KellyAH**
Change Ana [reccomended](https://github.com/zinc-collective/convene/issues/1472#issuecomment-1595647326) for [discussion](https://github.com/zinc-collective/convene/issues/1472#issuecomment-1595616304)  

- add note at the top of each doc mentioning that `Furniture` has been renamed to `Gizmo`
-  add note at the top mentioning that `Room` has been renamed to `Section`

Screenshot 2023-06-18 at 8 01 26 AM
Screenshot 2023-06-18 at 8 02 11 AM
Screenshot 2023-06-18 at 8 04 23 AM
Screenshot 2023-06-18 at 8 05 40 AM

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -32,6 +32,7 @@ Stakeholders for `Convene` fall into one or more of the following categories:
 - [`Operators`], who steward the [`Neighborhood`: Infrastructure], [`Neighborhood`: Trust & Safety], and [`Client`: Support] responsibilities
 - [`Contributors`] and [`Maintainers`], who design and build [`Furniture`].
 
+Note: `Furniture` has been renamed to `Gizmo/Gizmos`.
```
Comment (KellyAH) [inline CONTRIBUTING.md:36]: I had to add the note to a LOT of places. Maybe it's overkill. what to ya'll think?

Comment (anaulin) [reply CONTRIBUTING.md:36]: I think this is fine, especially since you already did it!

**Review by anaulin — APPROVED** [review]
Thank you for helping make our docs less confuzzling!

**Comment by KellyAH** [comment]
main merge CI: https://github.com/zinc-collective/convene/actions/runs/5305672321

**Comment by zspencer** [comment]
Thank you @KellyAH !!! This is great!

**Pull request merged**